src/config_file.cc

```diff
@@ -264,6 +280,9 @@ void CONFIG_TREC::setup_config( string filename ) {
   lmparam_smoothing1 = -1.0;
   lmparam_smoothing2 = -1.0;
   lmparam_mixture = -1.0;
+  normalizations = "ntn";
```

Comment:
All the other variables get initialised to an invalid value, but `normalizations` gets set to a default instead.

I think you can just set it to an empty string, i.e.:

```
normalizations = string();
```

And then the length check in `check_tfidf()` will `return false` if `normalizations` hasn't been set from the config file.